### ISSUE TITLE
Error when loading ONNX model

### ISSUE THREAD

**Issue by Anshul12256** [CalculatedContent/WeightWatcher#233]
Hello Everyone,
I was using the Weightwatcher package and trying to analyze my ONNX model. However, when I run the analyze command I get the following error:

```
Traceback (most recent call last):
  File "/home/linux/PycharmProjects/pythonProject/ww_analyze.py", line 62, in <module>
    details = watcher.analyze(model=model2, layers=[], min_evals=50, plot=True, randomize=True, mp_fit=True, savefig=True)
  File "/home/linux/.local/lib/python3.10/site-packages/weightwatcher/weightwatcher.py", line 2638, in analyze
    for ww_layer in layer_iterator:
  File "/home/linux/.local/lib/python3.10/site-packages/weightwatcher/weightwatcher.py", line 1049, in __next__
    return self.next()
  File "/home/linux/.local/lib/python3.10/site-packages/weightwatcher/weightwatcher.py", line 1052, in next
    curr_layer = next(self.layer_iter)
  File "/home/linux/.local/lib/python3.10/site-packages/weightwatcher/weightwatcher.py", line 1195, in ww_layer_iter_
    for curr_layer in self.model_iter:
  File "/home/linux/.local/lib/python3.10/site-packages/weightwatcher/weightwatcher.py", line 617, in layer_iter_
    yield ONNXLayer(model, inode, node)             
  File "/home/linux/.local/lib/python3.10/site-packages/weightwatcher/weightwatcher.py", line 575, in __init__
    the_type = self.layer_type(dims)
NameError: name 'dims' is not defined

Process finished with exit code 1
```
I would appreciate any suggestions to solve the error.

**Comment by charlesmartin14** [comment]
Can you share the model, even say with random weights ?

**Comment by Anshul12256** [comment]
Hello Sir,
Below is the drive link for downloading the ONNX models.

[https://drive.google.com/file/d/1sVIHx3k2prxHDltzFTmPg3gqzPIbzl8t/view?usp=sharing](url)

Thanks a lot and looking forward to your reply.

**Comment by charlesmartin14** [comment]
ok ill take a look

**Comment by charlesmartin14** [comment]
This has been fixed
Please try again with the version weightwatcher-0.7.1   on testpypi

 python3 -m pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple weightwatcher

**Comment by Anshul12256** [comment]
Thanks for fixing the issue. However, when I run the command I get the following output which means 0.7.1 version is not installed

```
Looking in indexes: https://test.pypi.org/simple/, https://pypi.org/simple
Requirement already satisfied: weightwatcher in ./ww/lib/python3.10/site-packages (0.7.0.9)
Requirement already satisfied: numpy in ./ww/lib/python3.10/site-packages (from weightwatcher) (1.24.2)
Requirement already satisfied: pandas in ./ww/lib/python3.10/site-packages (from weightwatcher) (2.0.0)
Requirement already satisfied: tqdm in ./ww/lib/python3.10/site-packages (from weightwatcher) (4.65.0)
Requirement already satisfied: matplotlib in ./ww/lib/python3.10/site-packages (from weightwatcher) (3.7.1)
Requirement already satisfied: powerlaw in ./ww/lib/python3.10/site-packages (from weightwatcher) (1.5)
Requirement already satisfied: matplotlib-inline in ./ww/lib/python3.10/site-packages (from weightwatcher) (0.1.6)
Requirement already satisfied: scikit-learn in ./ww/lib/python3.10/site-packages (from weightwatcher) (1.2.2)
Requirement already satisfied: pyparsing>=2.3.1 in ./ww/lib/python3.10/site-packages (from matplotlib->weightwatcher) (3.0.9)
Requirement already satisfied: fonttools>=4.22.0 in ./ww/lib/python3.10/site-packages (from matplotlib->weightwatcher) (4.39.3)
Requirement already satisfied: kiwisolver>=1.0.1 in ./ww/lib/python3.10/site-packages (from matplotlib->weightwatcher) (1.4.4)
Requirement already satisfied: pillow>=6.2.0 in ./ww/lib/python3.10/site-packages (from matplotlib->weightwatcher) (9.5.0)
Requirement already satisfied: python-dateutil>=2.7 in ./ww/lib/python3.10/site-packages (from matplotlib->weightwatcher) (2.8.2)
Requirement already satisfied: packaging>=20.0 in ./ww/lib/python3.10/site-packages (from matplotlib->weightwatcher) (23.0)
Requirement already satisfied: cycler>=0.10 in ./ww/lib/python3.10/site-packages (from matplotlib->weightwatcher) (0.11.0)
Requirement already satisfied: contourpy>=1.0.1 in ./ww/lib/python3.10/site-packages (from matplotlib->weightwatcher) (1.0.7)
Requirement already satisfied: traitlets in ./ww/lib/python3.10/site-packages (from matplotlib-inline->weightwatcher) (5.9.0)
Requirement already satisfied: tzdata>=2022.1 in ./ww/lib/python3.10/site-packages (from pandas->weightwatcher) (2023.3)
Requirement already satisfied: pytz>=2020.1 in ./ww/lib/python3.10/site-packages (from pandas->weightwatcher) (2023.3)
Requirement already satisfied: scipy in ./ww/lib/python3.10/site-packages (from powerlaw->weightwatcher) (1.10.1)
Requirement already satisfied: mpmath in ./ww/lib/python3.10/site-packages (from powerlaw->weightwatcher) (1.3.0)
Requirement already satisfied: threadpoolctl>=2.0.0 in ./ww/lib/python3.10/site-packages (from scikit-learn->weightwatcher) (3.1.0)
Requirement already satisfied: joblib>=1.1.1 in ./ww/lib/python3.10/site-packages (from scikit-learn->weightwatcher) (1.2.0)
Requirement already satisfied: six>=1.5 in ./ww/lib/python3.10/site-packages (from python-dateutil>=2.7->matplotlib->weightwatcher) (1.16.0)
```

**Comment by charlesmartin14** [comment]
Try it now.  Should be at weightwatcher-0.7.1.1

**Comment by charlesmartin14** [comment]
Closing this issue; bugs fixed in 0.7.1
